### PR TITLE
Adds a "paths" command to loom.

### PR DESCRIPTION
**Pull request by damncabbage**
... to dump out files for `purs` / `purs.test` bundles.

The implementation is currently stubbed. I've been chasing down how to pull out the needed locations, but this is a separate chunk that's reviewable while I do that.

! @sphvn @charleso 
/jury approved @charleso

### CODE REVIEW

File: loom-cli/main/loom.hs [hunk 1]
```diff
@@ -250,6 +280,53 @@ siteFilterP =
...
+    <*> pathGroupingP
+
+parseBundle :: Text -> Either Text HardcodedBundle
+parseBundle = \case
+  "purs" ->
+    pure Purs
+  "purs.test" ->
+    pure PursTest
+  _ ->
+    Left "Unsupported bundle; only available for 'purs' and 'purs.test'"
```
Comment (charleso) [inline loom-cli/main/loom.hs]: FWIW you can just use subparser and command' if you want the help for free

File: loom-cli/main/loom.hs [hunk 2]
```diff
@@ -250,6 +280,53 @@ siteFilterP =
...
+        )
+  <|> pure AllPaths
+
+pathGroupingP :: Parser PathGrouping
+pathGroupingP =
+      OA.flag' AsGlobs (
+           OA.long "as-globs"
+        <> OA.help "Display as path globs (default)"
+        )
+  <|> OA.flag' AsGlobs (
```
Comment (charleso) [inline loom-cli/main/loom.hs]: `AsFiles`?

**Comment by charleso** [comment]
👍 

Feel free to add to the [cli test](https://github.com/ambiata/loom/blob/master/loom-cli/test/cli/purs/run) too, which is handy for documenting/tracking what you expect

File: loom-cli/main/loom.hs [hunk 3]
```diff
@@ -111,6 +137,8 @@ main = do
             Exit.exitWith ec
       Watch port sf ->
         watch port sf
+      Paths (PathsCommand _ _ _) ->
+        IO.hPutStrLn IO.stderr "TODO"
```
Comment (charleso) [inline loom-cli/main/loom.hs]: 💯  :)

File: loom-cli/main/loom.hs (hunk 2 above)
Comment (damncabbage) [reply loom-cli/main/loom.hs]: Ah, yes; oops.

File: loom-cli/main/loom.hs (hunk 1 above)
Comment (damncabbage) [reply loom-cli/main/loom.hs]: I'm not entirely sure how to structure this without going another level of nesting; might have a chat when you're free, if you don't mind.

**Comment by damncabbage** [comment]
> Feel free to add to the cli test too, which is handy for documenting/tracking what you expect

I'm definitely going to be using this, but right now it's only returning TODO. I'll be using it (with the purs sample project) to check the actual results.

**Comment by sphvn** [comment]
:+1:

**Pull request merged**